config/environment.js

```diff
@@ -28,7 +28,7 @@ module.exports = function(environment) {
     },
 
     APP: {
-      apiHost      : process.env.API_HOST || 'https://open-event-api.herokuapp.com',
+      apiHost      : process.env.API_HOST || 'http://localhost:5000',
```

Comment:
@harshitagupta30 Currently I am getting an error which is not related to this. Will do it once I resolve other.